tests/test_series.py

```diff
@@ -3285,9 +3286,45 @@ def callable(x: int | NAType) -> str | NAType:
     check(
         assert_type(s.map(callable, na_action=None), "pd.Series[str]"), pd.Series, str
     )
+    check(assert_type(s.map(callable), "pd.Series[str]"), pd.Series, str)
 
     series = pd.Series(["a", "b", "c"])
     check(assert_type(s.map(series, na_action=None), "pd.Series[str]"), pd.Series, str)
+    check(assert_type(s.map(series), "pd.Series[str]"), pd.Series, str)
+
+    s2: pd.Series[float] = pd.Series([1.0, pd.NA, 3.0])
+
+    def callable2(x: float) -> float:
+        return x + 1
+
+    check(
+        assert_type(s2.map(callable2, na_action="ignore"), "pd.Series[float]"),
+        pd.Series,
+        float,
+    )
+    check(
+        assert_type(s2.map(callable2), "pd.Series[float]"),
+        pd.Series,
+        float,
+    )
+    if TYPE_CHECKING_INVALID_USAGE:
+        s2.map(callable2, na_action=None)  # type: ignore[arg-type] # pyright: ignore[reportCallIssue, reportArgumentType]
+
+    s3: pd.Series[str] = pd.Series(["A", pd.NA, "C"])
+
+    def callable3(x: str) -> str:
```

Comment:
Non-numerical functions pass only the explicit `na_action="ignore"` but not the `na_action=None` case (explicit or implicit) because these are likely to cause runtime errors on nan-values.